pandas/_libs/src/datetime/pd_datetime.c

```diff
@@ -245,7 +245,10 @@ static int pandas_datetime_exec(PyObject *Py_UNUSED(module)) {
 }
 
 static PyModuleDef_Slot pandas_datetime_slots[] = {
-    {Py_mod_exec, pandas_datetime_exec}, {0, NULL}};
+    {Py_mod_exec, pandas_datetime_exec},
+    {Py_mod_gil, Py_MOD_GIL_NOT_USED},
```

Comment:
As @WillAyd mentioned, this is new in 3.13:

```
#if PY_VERSION_HEX >= 0x030D0000
    {Py_mod_gil, Py_MOD_GIL_NOT_USED},
#endif
```

https://docs.python.org/3.14/howto/free-threading-extensions.html#multi-phase-initialization

Reply:
Thanks for this. Friday was not a good day, apparently. 🤕 

One question: In the [free-threaded repo's README](https://github.com/Quansight-Labs/free-threaded-compatibility?tab=readme-ov-file#running-python-with-the-gil-disabled) we recomment guarding with `#ifdef Py_GIL_DISABLED`. Should we change that?

Reply:
I haven't kept tabs on the CPython implementation but seems like what the free-threaded repo suggests might be better than checking the version hex - there will be both GIL and GIL-less Python compilations for a few versions right?

Reply:
@WillAyd That's correct, however `Py_mod_gil` is just ignored in 3.13+ non-free-threaded builds. Effectively, both guard lead to correct behavior.

Reply:
As a matter of style I think checking for `Py_GIL_DISABLED` would make more sense if you were evaluating whether that was true/false, not just checking if the symbol was defined. 

So happy with what we have here, but probably worth asking for input from other projects for the free-threaded repo standard.

We can merge this as is for now and follow up with a different approach if a more canonical one becomes clearer